core/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -199,7 +199,7 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
     OptimizeTableCommand(
       Option(ctx.path).map(string),
       Option(ctx.table).map(visitTableIdentifier),
-      Option(ctx.partitionPredicate).map(extractRawText(_)), Map.empty)(interleaveBy)
+      Seq(Option(ctx.partitionPredicate).map(extractRawText(_))).flatten, Map.empty)(interleaveBy)
```

Comment:
is there any better way of doing this ?

Reply:
`Option(ctx.partitionPredicate).map(extractRawText(_)).toSeq`?

Reply:
\* quietly closing this review *